[2025-03-03T13:14:20.826-0600] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-03T13:14:20.861-0600] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: EtlWeather.WeatherAPI manual__2025-03-03T19:14:14.292439+00:00 [queued]>
[2025-03-03T13:14:20.881-0600] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: EtlWeather.WeatherAPI manual__2025-03-03T19:14:14.292439+00:00 [queued]>
[2025-03-03T13:14:20.882-0600] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-03T13:14:20.917-0600] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): WeatherAPI> on 2025-03-03 19:14:14.292439+00:00
[2025-03-03T13:14:20.928-0600] {standard_task_runner.py:72} INFO - Started process 30757 to run task
[2025-03-03T13:14:20.948-0600] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'EtlWeather', 'WeatherAPI', 'manual__2025-03-03T19:14:14.292439+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/dataEngineering.py', '--cfg-path', '/tmp/tmp62c2380q']
[2025-03-03T13:14:20.953-0600] {standard_task_runner.py:105} INFO - Job 64: Subtask WeatherAPI
[2025-03-03T13:14:21.102-0600] {task_command.py:467} INFO - Running <TaskInstance: EtlWeather.WeatherAPI manual__2025-03-03T19:14:14.292439+00:00 [running]> on host IMNOTROY.
[2025-03-03T13:14:21.293-0600] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='EtlWeather' AIRFLOW_CTX_TASK_ID='WeatherAPI' AIRFLOW_CTX_EXECUTION_DATE='2025-03-03T19:14:14.292439+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-03T19:14:14.292439+00:00'
[2025-03-03T13:14:21.297-0600] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-03T13:14:21.956-0600] {dataEngineering.py:28} INFO - Data gotten succesfully!
[2025-03-03T13:14:21.957-0600] {python.py:240} INFO - Done. Returned value was: {'dt': 1741029262, 'sunrise': 1741006466, 'sunset': 1741048961, 'temp': 299.69, 'feels_like': 299.69, 'pressure': 1013, 'humidity': 25, 'dew_point': 278.08, 'uvi': 11.98, 'clouds': 20, 'visibility': 10000, 'wind_speed': 4.47, 'wind_deg': 80, 'wind_gust': 3.13, 'weather': [{'id': 801, 'main': 'Clouds', 'description': 'few clouds', 'icon': '02d'}]}
[2025-03-03T13:14:22.143-0600] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-03T13:14:22.143-0600] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=EtlWeather, task_id=WeatherAPI, run_id=manual__2025-03-03T19:14:14.292439+00:00, execution_date=20250303T191414, start_date=20250303T191420, end_date=20250303T191422
[2025-03-03T13:14:22.246-0600] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-03T13:14:22.286-0600] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-03T13:14:22.297-0600] {local_task_job_runner.py:245} INFO - ::endgroup::
